src/nspawn/nspawn.c

```diff
@@ -187,6 +187,11 @@ static SettingsMask arg_settings_mask = 0;
 static int arg_settings_trusted = -1;
 static char **arg_parameters = NULL;
 static const char *arg_container_service_name = "systemd-nspawn";
+/* nspawn is listening on the socket at the path in the constant nspawn_notify_socket_path
+ * nspawn_notify_socket_path is relative to the container
+ * the init process in the container pid can send messages to nspawn following the sd_notify(3) protocol */
+static const char *nspawn_notify_socket_path = "/run/systemd/nspawn/notify";
```

Comment:
why is this configurable? Just make this a `#define` or even better, not even that, just use the literal string at the few places this is needed, as there really shouldn't be a reason to reconfigure this path anyway...